xml/security_ldap_ca.xml

```diff
@@ -82,8 +82,8 @@
    Now import the <replaceable>SERVER.p12</replaceable> file
    into your &ds389a; NSS database:
   </para>
-  <screen>&prompt.sudo;<command>dsctl <replaceable>INSTANCE_NAME</replaceable> tls remove-cert Self-Signed-CA   pk12util -i <replaceable>SERVER.p12</replaceable></command> \
-<command>-d /etc/dirsrv/slapd-<replaceable>INSTANCE-NAME</replaceable>/cert9.db</command></screen>
+  <screen>&prompt.sudo;<command>dsctl <replaceable>INSTANCE_NAME</replaceable> tls remove-cert Self-Signed-CA \
```

Comment:
There is a `;` missing before the `\` otherwise the two concatenated lines will still be one command.